### PR TITLE
Setwifi

### PR DESCRIPTION
**Pull request by yozhgoor**
i have add a panel who display the captive portal in the Wifi settings

### CODE REVIEW

**Comment by yozhgoor** [comment]
we can now close the captive-portal dialog and we can return on this dialog in settings > advanced settings > wifi settings

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,7 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# text_editor
+src/third-i.sublime-project
+src/third-i.sublime-workspace
```
Comment (cecton) [inline .gitignore]:
```suggestion
```

File: package-lock.json [hunk 2]
```diff
@@ -2757,6 +2757,15 @@
...
     },
+    "bindings": {
+      "version": "1.5.0",
+      "resolved": "https://registry.npmjs.org/bindings/-/bindings-1.5.0.tgz",
+      "integrity": "sha512-p2q/t/mhvuOj/UeLlV6566GD/guowlr0hHxClI0W9m7MWYkL1F0hLo+0Aexs9HSPCtR1SXQ0TD3MMKrXZajbiQ==",
+      "optional": true,
+      "requires": {
+        "file-uri-to-path": "1.0.0"
+      }
+    },
```
Comment (cecton) [inline package-lock.json]:
```suggestion
```

File: package-lock.json [hunk 3]
```diff
@@ -5524,6 +5533,12 @@
         "schema-utils": "^2.5.0"
       }
     },
+    "file-uri-to-path": {
+      "version": "1.0.0",
+      "resolved": "https://registry.npmjs.org/file-uri-to-path/-/file-uri-to-path-1.0.0.tgz",
+      "integrity": "sha512-0Zt+s3L7Vf1biwWZ29aARiVYLx7iMGnEUl9x33fbB/j3jR81u/O2LbqK+Bm1CDSNDKVtJ/YjwY7TUd5SkeLQLw==",
+      "optional": true
+    },
```
Comment (cecton) [inline package-lock.json]:
```suggestion
```

File: package-lock.json [hunk 4]
```diff
@@ -7104,7 +7119,11 @@
           "version": "1.2.13",
           "resolved": "https://registry.npmjs.org/fsevents/-/fsevents-1.2.13.tgz",
           "integrity": "sha512-oWb1Z6mkHIskLzEJ/XWX0srkpkTQ7vaopMQkyaEIoq0fmtFVxOthb8cCxeT+p3ynTdkk/RZwbgG4brR5BeWECw==",
-          "optional": true
+          "optional": true,
+          "requires": {
+            "bindings": "^1.5.0",
+            "nan": "^2.12.1"
+          }
```
Comment (cecton) [inline package-lock.json:7126]:
```suggestion
```

Comment (cecton) [inline package-lock.json]:
```suggestion
          "optional": true
```

File: package-lock.json [hunk 5]
```diff
@@ -8179,6 +8198,12 @@
       "resolved": "https://registry.npmjs.org/mute-stream/-/mute-stream-0.0.8.tgz",
       "integrity": "sha512-nnbWWOkoWyUsTjKrhgD0dcz22mdkSnpYqbEjIm2nhwhuxlSkpywJmBo8h0ZqJdkp73mb90SssHkN4rsRaBAfAA=="
     },
+    "nan": {
+      "version": "2.14.1",
+      "resolved": "https://registry.npmjs.org/nan/-/nan-2.14.1.tgz",
+      "integrity": "sha512-isWHgVjnFjh2x2yuJ/tj3JbwoHu3UC2dX5G/88Cm24yB6YopVgxvBObDY7n5xW6ExmFhJpSEQqFPvq9zaXc8Jw==",
+      "optional": true
+    },
```
Comment (cecton) [inline package-lock.json]:
```suggestion
```

File: package-lock.json [hunk 6]
```diff
@@ -12742,7 +12767,11 @@
           "version": "1.2.13",
           "resolved": "https://registry.npmjs.org/fsevents/-/fsevents-1.2.13.tgz",
           "integrity": "sha512-oWb1Z6mkHIskLzEJ/XWX0srkpkTQ7vaopMQkyaEIoq0fmtFVxOthb8cCxeT+p3ynTdkk/RZwbgG4brR5BeWECw==",
-          "optional": true
+          "optional": true,
+          "requires": {
+            "bindings": "^1.5.0",
+            "nan": "^2.12.1"
+          }
```
Comment (cecton) [inline package-lock.json]:
```suggestion
```

File: package-lock.json [hunk 7]
```diff
@@ -13055,7 +13084,11 @@
...
```
Comment (cecton) [inline package-lock.json:13091]:
```suggestion
```

Comment (cecton) [inline package-lock.json]:
```suggestion
          "optional": true
```

File: package-lock.json [hunk 8]
```diff
@@ -12742,7 +12742,6 @@
           "version": "1.2.13",
           "resolved": "https://registry.npmjs.org/fsevents/-/fsevents-1.2.13.tgz",
           "integrity": "sha512-oWb1Z6mkHIskLzEJ/XWX0srkpkTQ7vaopMQkyaEIoq0fmtFVxOthb8cCxeT+p3ynTdkk/RZwbgG4brR5BeWECw==",
-          "optional": true
         },
```
Comment (cecton) [inline package-lock.json]:
```suggestion
          "optional": true
        },
```

**Pull request merged**